Mail/Commands/NewWindowCommand.swift

```diff
@@ -0,0 +1,45 @@
+/*
+ Infomaniak Mail - iOS App
+ Copyright (C) 2024 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import InfomaniakCore
+import InfomaniakCoreUI
+import InfomaniakDI
+import MailCore
+import MailResources
+import SwiftUI
+
+@available(iOS 16.0, *)
+struct NewWindowCommand: View {
+    @LazyInjectService private var matomo: MatomoUtils
+
+    @Environment(\.openWindow) private var openWindow
+
+    let rootViewState: RootViewState
+
+    var body: some View {
+        Button(MailResourcesStrings.Localizable.buttonNewWindow) {
+            newWindow(rootViewState: rootViewState)
+        }
+        .keyboardShortcut("n", modifiers: [.option, .command])
+    }
+
+    @MainActor func newWindow(rootViewState: RootViewState) {
```

Comment:
Why do we want to have the rootViewState as a function parameter, given that it is already a property of the struct?

Reply:
Fixed